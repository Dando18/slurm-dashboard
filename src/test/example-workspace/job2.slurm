$!/bin/bash
#SBATCH -t 00:01:00

echo "test"
