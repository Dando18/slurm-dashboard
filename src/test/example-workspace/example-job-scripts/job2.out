job 2 output
